server/viame_server/viame.py

```diff
@@ -52,6 +53,21 @@ def __init__(self):
         self.route("DELETE", ("attribute", ":id"), self.delete_attribute)
         self.route("GET", ("valid_images",), self.get_valid_images)
 
+    @access.public
+    @autoDescribeRoute(Description("Get custom brand data"))
+    def get_brand_data(self):
+        adminUserIds = [user['_id'] for user in User().getAdmins()]
+        # Find an item owned by an admin with meta.brand=True
+        data = Item().findOne(
+            {
+                'meta.brand': True,
```

Comment:
may want a more lenient way to test for meta.brand.  The GUI editor for simple will make it a 'true' string instead of true.  While I like the JSON editor may not be the best to force someone into it (even if we are talking about admins). May want this to be truthy instead of looking for `true===true`.

Reply:
Heh, I noticed this myself.  Agree a simple truthiness check would be fine.